pkg/fqdn/cache.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fqdn
+
+import (
+	"bytes"
+	"net"
+	"sort"
+	"time"
+
+	"github.com/cilium/cilium/pkg/lock"
+)
+
+var DefaultDNSCache = NewDNSCache()
+
+// cacheEntry objects are immutable once created.
+// They hold an the DNS data passed in with DNSCache.Update, nominally equating
+// to a DNS lookup. cacheEntry objects are managed by DNSCache and are not
```

Comment:
and are --> are

Reply:
I ended up shuffling this around. I wanted to express that they were an internal type that should not leak out of DNSCache. I just say that directly now :)